### DISCUSSION TITLE
Same challenge hash for all 64 counts

### DISCUSSION THREAD

**Discussion by nomadengineer** [Chia-Network/chia-blockchain#3705] [Q&A]
Hello, I noticed something that the challenge hash remains the same for 64 counts of the challenge and changes when the count starts from 0. is this normal?

![image](https://user-images.githubusercontent.com/45670423/117042248-26b93800-ad25-11eb-997a-84a6caeb487d.png)

**Comment by cccat6** [comment]
Yes, it's normal. No worry
